src/jdk.compiler/share/man/javac.md

```diff
@@ -1285,6 +1285,12 @@ although some module-related path options allow a package hierarchy to be
 specified on a per-module basis. All other path options are used to specify
 package hierarchies.
 
+When a JAR file in the user class path has a `Class-Path` manifest attribute,
+and the specified JAR file(s) exist, they are automatically inserted into the
+user class path after the JAR file. This rule also applies recursively to any
+new JAR files found. Consult the [JAR File Specification](../jar/jar.html#class-path-attribute)
```

Comment:
The link to the `jar.html` file using the relative path `../jar/` should lead to the JAR File Specification document. For JavaSE 23, that specification documentation is located at https://docs.oracle.com/en/java/javase/23/docs/specs/jar/jar.html

How can I test that the online version, for 23 hosted at https://docs.oracle.com/en/java/javase/23/docs/specs/man/javac.html, contains the correct link target?